apps/website/package.json

```diff
@@ -104,7 +103,11 @@
     "@types/react": "^17.0.0"
   },
   "browserslist": {
-    "production": [">0.5%", "not dead", "not op_mini all"],
+    "production": [
+      ">0.5%",
+      "not dead",
+      "not op_mini all"
+    ],
```

Comment:
this is becoming a problem, prettier and lerna or some script keep re-formatting json documents.

Reply:
I disabled them. They will only run manually now.

Reply:
This new formatting is what VS code does automatically. We should check it in. It shouldn't auto change any more.